antarest/core/config.py

```
@@ -300,6 +303,9 @@ def from_dict(cls, data: JSON) -> "LocalConfig":
         Returns: object NbCoresConfig
         """
         defaults = cls()
+        _id = data.get("id", "id")
```

All those field should be mandatory and not have a default value:
id, name, type. If one is absent, we should raise an InvalidConfigurationError